### PR TITLE
Add Tutorial Content Service

### PR DESCRIPTION
**Pull request by benjimarshall**
### Changes
- Add service to get the contents of a given tutorial for that user.
- Add tests for this service.

### Testing
Add unit tests for the service, all tests pass.

### CODE REVIEW

File: DigitalLearningSolutions.Data.Tests/Services/TutorialContentServiceTests.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+        [SetUp]
+        public void Setup()
+        {
+            var connection = ServiceTestHelper.GetDatabaseConnection();
+            tutorialContentService = new TutorialContentService(connection);
+        }
+
+        [Test]
+        public void Get_tutorial_content_of_consistent_tutorial_should_return_tutorial()
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Data.Tests/Services/TutorialContentServiceTests.cs:21]: What do you mean by "consistent"? 🙂

File: DigitalLearningSolutions.Data/Models/TutorialContent/TutorialContent.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
+﻿namespace DigitalLearningSolutions.Data.Models.TutorialContent
+{
+    public class TutorialContent
+    {
+        public int Id { get; }
+        public int CandidateID { get; }
+        public int SectionID { get; }
+        public int CustomisationID { get; }
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Data/Models/TutorialContent/TutorialContent.cs:8]: Do we need the candidate id, section id and customisation id? 🙂

File: DigitalLearningSolutions.Data/Models/TutorialContent/TutorialContent.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        public int TimeSpent { get; }
+        public int AverageTutMins { get; }
+        public int TutScore { get; }
+        public int PossScore { get; }
+        public bool DiagStatus { get; }
+        public int DiagAttempts { get; }
+        public string Objectives { get; }
+        public string VideoPath { get; }
+        public string TutorialPath { get; }
+        public string SupportingMatsPath { get; }
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Data/Models/TutorialContent/TutorialContent.cs:20]: We don't need to match the names in the old system, so probably worth making some of these names more descriptive? 🙂

**Review by FridaTveit — COMMENTED** [review]
A few comments, otherwise looks good 🙂

File: DigitalLearningSolutions.Data.Tests/Services/TutorialContentServiceTests.cs (hunk 1 above)
Comment (benjimarshall) [reply DigitalLearningSolutions.Data.Tests/Services/TutorialContentServiceTests.cs:21]: The tutorial ID exists on that customisation and section, and there is an `aspProgress` entry for this tutorial ID and candidate ID. Maybe this test could be renamed to explain this? I can't think of any concise ways of saying this, do you have an suggestions? 🙂

Comment (FridaTveit) [reply DigitalLearningSolutions.Data.Tests/Services/TutorialContentServiceTests.cs:21]: Maybe having a few more tests:
* `Get_tutorial_content_should_return_tutorial()`
* `Get_tutorial_content_should_return_null_if_tutorial_id_invalid()`
* Same as above but customisation id invalid
* Same as above but section id invalid

What do you think? 🙂

File: DigitalLearningSolutions.Data/Models/TutorialContent/TutorialContent.cs (hunk 2 above)
Comment (benjimarshall) [reply DigitalLearningSolutions.Data/Models/TutorialContent/TutorialContent.cs]: Probably not, all of this information could be extracted from the route variables. Refactored in 53f828d 🙂

File: DigitalLearningSolutions.Data/Models/TutorialContent/TutorialContent.cs (hunk 3 above)
Comment (benjimarshall) [reply DigitalLearningSolutions.Data/Models/TutorialContent/TutorialContent.cs]: Yep, I had meant to ask whether I could give them better names! Refactored in 53f828d 🙂

File: DigitalLearningSolutions.Data.Tests/Services/TutorialContentServiceTests.cs (hunk 1 above)
Comment (benjimarshall) [reply DigitalLearningSolutions.Data.Tests/Services/TutorialContentServiceTests.cs]: I like it! Done in bd57872 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**